.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
```

Comment:
I'm surprised this works — `on` is a "truthy" value in YAML and should get treated like `true` when it's not in quotes...

```
Python 3.7.7 (default, Mar 10 2020, 15:43:33) 
[Clang 11.0.0 (clang-1100.0.33.17)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import yaml, sys
>>> yaml.dump(yaml.load("on: [push, pull_request]\n"), stream=sys.stdout)
true: [push, pull_request]
>>> yaml.dump(yaml.load("'on': [push, pull_request]\n"), stream=sys.stdout)
'on': [push, pull_request]
```

However, all of GitHub's documentation shows `on` being used unquoted in workflow YAML files, so 🤷‍♂️...